examples/wasm/README.md

```diff
@@ -1,86 +1,76 @@
-In this example, we show how a Lua filter can be used with the Envoy
-proxy. The Envoy proxy [configuration](./envoy.yaml) includes a lua
-filter that contains two functions namely
-`envoy_on_request(request_handle)` and
-`envoy_on_response(response_handle)` as documented
-[here](https://www.envoyproxy.io/docs/envoy/latest/configuration/http_filters/lua_filter).
+# Envoy WebAssembly Filter
 
+In this example, we show how a WebAssembly(WASM) filter can be used with the Envoy
+proxy. The Envoy proxy [configuration](./envoy.yaml) includes a Webassembly filter
+as documented [here](https://www.envoyproxy.io/docs/envoy/latest/).
+<!--TODO(bianpengyuan): change to the url of Wasm filter once the doc is ready.-->
 
 
-# Usage
-1. `docker-compose build`
-2. `docker-compose up`
-3. `curl -v localhost:8000`
 
-## Sample Output:
+## Quick Start
+
+1. `docker-compose build`
+1. `docker-compose up`
+1. `curl -v localhost:18000`
```

Comment:
Nit: why all list elements start with `1.` in this file? I know it's generated properly on GitHub, but it's a bit annoying when looking at the raw file.